spring-cloud-modules/spring-cloud-contract/spring-cloud-contract-consumer/pom.xml

```diff
@@ -40,6 +40,16 @@
             <version>${project.parent.version}</version>
             <scope>test</scope>
         </dependency>
```

Comment:
As rest-assured is inherited and brings old groovy, exclude that groovy version (2.5.6)